pkgs/applications/misc/ArchiSteamFarm/update.sh

```diff
@@ -1,10 +1,8 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -I nixpkgs=../../../.. -i bash -p curl gnused jq common-updater-scripts
```

Comment:
Using the local nixpkgs checkout seems valuable, in case that contains necessary updates not present in the channels yet.

 I also recall the maintainer script having trouble when not executed from nixpkgs root, but I'm not entirely sure about that.

Reply:
The problem is that you then always need to cd into the directory where the update script is.